azext_edge/edge/providers/check/dataflow.py

```diff
@@ -53,9 +61,13 @@
 dataflow_profile_check_name = "evalDataflowProfiles"
 dataflow_profile_check_desc = "Evaluate Dataflow Profiles"
 
+registry_endpoint_check_name = "evalRegistryEndpoints"
+registry_endpoint_check_desc = "Evaluate Registry Endpoints"
+
 dataflow_target = "dataflows.connectivity.iotoperations.azure.com"
 dataflow_endpoint_target = "dataflowendpoints.connectivity.iotoperations.azure.com"
 dataflow_profile_target = "dataflowprofiles.connectivity.iotoperations.azure.com"
+registry_endpoint_target = "registryendpoints.connectivity.iotoperations.azure.com"
```

Comment:
registry endpoints are also used for akri connector templates

Reply:
Are they shared or different?